tests/validate/bandwidth_test/src/host.cpp

```diff
@@ -66,7 +66,8 @@ int main(int argc, char** argv) {
         }
     }
 
-    int NUM_KERNEL;
+    int NUM_KERNEL = 0, NUM_KERNEL_DDR = 0;
```

Comment:
Please use lower case variable names.  UPPER CASE is reserved for macros (which should not really ever be needed) and would need to be scoped by unique prefix.  Lower case is safe from macro substitution.